docker/cassandra/Dockerfile

```diff
@@ -19,6 +23,7 @@ COPY cql-readiness-probe /usr/bin/cql-readiness-probe
 COPY ${cassandra_k8s_addons_jar} /usr/share/cassandra/lib
 
 ADD default-config /etc/cassandra
+RUN chown -R cassandra:cassandra /etc/cassandra
 
 VOLUME /var/lib/cassandra
```

Comment:
What about permissions on the `/var/lib/cassandra` volume? Is the default owner root?

Reply:
No, they are owned by `cassandra` already (i.e. even before this change):

```
root@0ff4dd9f253d:/# ls -alh /var/lib/ | grep cas
drwxr-xr-x  2 cassandra cassandra 4.0K Jul  2  2018 cassandra
```